Extensions/BitBucket/Src/Tasks/DownloadArtifactsBitbucket/task.json

```diff
@@ -13,7 +13,7 @@
   "demands": [],
   "version": {
     "Major": 15,
-    "Minor": 263,
+    "Minor": 264,
     "Patch": 1
```

Comment:
We should reset patch to `0`, since minor is updated. Let's do the same for vss-extension.json